src/main/java/org/jabref/gui/customentrytypes/CustomizeEntryTypeDialogView.java

```diff
@@ -141,6 +141,7 @@ private void setupTable() {
         //Here we would need to select the
         EasyBind.subscribe(viewModel.selectedEntryTypeProperty(), type -> {
             if (type != null) {
+                fields.itemsProperty().unbindBidirectional(type.fields());
```

Comment:
@tobiasdiez I followed your approach, however, sometimes the field list  on the right is not updated.
I tried unbinding/binding again but doesn't help. Any idea  how to fix this?

Reply:
This should be `fields.setItems(type.fields())` after converting all the above list properties to observable lists.

Moreover, I would put this subscription thing in the `viewModel` and expose a new list `viewModel.fieldsForSelectedType`.